Assets/Scripts/Interaction/Interactable_StartQuest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Interaction
+{
+    public class Interactable_StartQuest : Interactable
+    {
+        public QuestHandler questHandler;
+        public Questlog questlog;
+        public QuestRepository questRepository;
+        public int questId;
+        public override void Interact(Interactor interactor )
+        {
+            foreach (Quest q in questlog.quests)
+            {
+                if (q.questId == questId)
+                {
+                    return;
+                }
+            }
```

Comment:
can you move this code to the questhandler, it doesnt belong in the interactable.
maybe name it trystartquest.